api/lib/oauth/oauth.js

```diff
@@ -69,6 +69,58 @@ module.exports = function(configFilePath) {
 
   }
 
+  obj.checkAutoScalerBinding = function(req, callback) {
+    var appId = req.params.app_id;
+    if (!appId) {
+      logger.error("Failed to get appId");
+      callback({ "statusCode": HttpStatus.NotFound, "message": "Failed to get appId" }, null);
+      return;
+    }
+    var ccEndpoint = getCloudControllerEndpoint(req);
+    var userToken = req.header("Authorization");
+    if (!userToken) {
+      logger.info("User token is not provided");
+      callback(null, false);
+      return;
+    }
+    var options = {
+      url: ccEndpoint + "/v2/apps/" + appId + "/env",
+      method: "GET",
+      json: true,
+      timeout: 10000,
+      headers: {
+        "Authorization": userToken,
+        "Content-Type": "application/json"
+      }
+    };
+    request(options, function(error, response, body) {
```

Comment:
why not checking it from our own database?

Reply:
By checking from database, it will add broker's database as dependency of apiserver.